### PR TITLE
Fix #9381 image featureinfo

### LINKED ISSUE #9381: Small fix for Images management in Identify template
## Description

1. The undefined text should be removed as requested.
Image

2. There should be an indication in the _add image_ tool that also a placeholder can be specified in order to get the URL from a feature attribute (maybe below the URL field). This would help a lot the user to understand how the tool can work. For example something like the one below.
Image

3. Is there a way to change the icon of broken image using a better one with some explanatory text inside? For example something like the one below.
Image

4. **Question:** is there a way using the tool to modify an inserted URL value (direct value or placeholder)? This possibility is usually present in tools like this and it should present be also there

- [x] Not browser related

<details><summary> <b>Browser info</b> </summary>

(use this site: <a href="https://www.whatsmybrowser.org/">https://www.whatsmybrowser.org/</a> for non expert users)

| Browser Affected | Version |
|---|---|
|Internet Explorer| |
|Edge| |
|Chrome| |
|Firefox| |
|Safari| |
</details>

### PR DESCRIPTION
**Pull request by MV88**
## Description

this pr adds guide text for the image tool in feature info editor and manage it outside the editor (if the src is not reachable it shows an image like this)
image

Editing the internal html rendered is not easy connecting the rendered image to the toolbar tool that has local state

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

Fix #9381 

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: web/client/translations/data.en-US.json [hunk 1]
```diff
@@ -91,6 +91,8 @@
             "groupProperties": "Group properties",
             "featureInfo": "Feature Info",
             "featureInfoFormatLbl": "Identify response format",
+            "guideText": "Supports Urls or Base64URI. Use a placeholder like $\\{properties.YOUR_ATTRIBUTE\\} in order to fetch the url from a feature attribute",
```
Comment (offtherailz) [inline web/client/translations/data.en-US.json:94]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URLs Data URL or DATA URI is the offical name (data URI is more correct in my opinion, even if mozilla provide it as data URL).

Base64 URL is only an abbreviation we use internally, but you can not understand it this way.

I tried this Data URI and effectively it is not supported. So it seems that only binary base64 images are supported ( I verified also on dev).

```
data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="0px" y="0px" viewBox="0 0 100 100" enable-background="new 0 0 100 100" xml:space="preserve" height="100px" width="100px"><g><path d="M28.1,36.6c4.6,1.9,12.2,1.6,20.9,1.1c8.9-0.4,19-0.9,28.9,0.9c6.3,1.2,11.9,3.1,16.8,6c-1.5-12.2-7.9-23.7-18.6-31.3 c-4.9-0.2-9.9,0.3-14.8,1.4C47.8,17.9,36.2,25.6,28.1,36.6z"/><path d="M70.3,9.8C57.5,3.4,42.8,3.6,30.5,9.5c-3,6-8.4,19.6-5.3,24.9c8.6-11.7,20.9-19.8,35.2-23.1C63.7,10.5,67,10,70.3,9.8z"/><path d="M16.5,51.3c0.6-1.7,1.2-3.4,2-5.1c-3.8-3.4-7.5-7-11-10.8c-2.1,6.1-2.8,12.5-2.3,18.7C9.6,51.1,13.4,50.2,16.5,51.3z"/><path d="M9,31.6c3.5,3.9,7.2,7.6,11.1,11.1c0.8-1.6,1.7-3.1,2.6-4.6c0.1-0.2,0.3-0.4,0.4-0.6c-2.9-3.3-3.1-9.2-0.6-17.6   c0.8-2.7,1.8-5.3,2.7-7.4c-5.2,3.4-9.8,8-13.3,13.7C10.8,27.9,9.8,29.7,9,31.6z"/><path d="M15.4,54.7c-2.6-1-6.1,0.7-9.7,3.4c1.2,6.6,3.9,13,8,18.5C13,69.3,13.5,61.8,15.4,54.7z"/><path d="M39.8,57.6C54.3,66.7,70,73,86.5,76.4c0.6-0.8,1.1-1.6,1.7-2.5c4.8-7.7,7-16.3,6.8-24.8c-13.8-9.3-31.3-8.4-45.8-7.7   c-9.5,0.5-17.8,0.9-23.2-1.7c-0.1,0.1-0.2,0.3-0.3,0.4c-1,1.7-2,3.4-2.9,5.1C28.2,49.7,33.8,53.9,39.8,57.6z"/><path d="M26.2,88.2c3.3,2,6.7,3.6,10.2,4.7c-3.5-6.2-6.3-12.6-8.8-18.5c-3.1-7.2-5.8-13.5-9-17.2c-1.9,8-2,16.4-0.3,24.7   C20.6,84.2,23.2,86.3,26.2,88.2z"/><path d="M30.9,73c2.9,6.8,6.1,14.4,10.5,21.2c15.6,3,32-2.3,42.6-14.6C67.7,76,52.2,69.6,37.9,60.7C32,57,26.5,53,21.3,48.6   c-0.6,1.5-1.2,3-1.7,4.6C24.1,57.1,27.3,64.5,30.9,73z"/></g></svg>
```

So I explained a little better in suggestion. Please translate. 

```suggestion
            "guideText": "In the URL of the image you can use placeholders like $\\{properties.YOUR_ATTRIBUTE\\} to make the URL parametric, depending by the feature attributes. It supports both URLs or \"Data URIs\" encoded in base64",
```


Here I attach an svg image encoded in base64 that works

```
data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8+CjwhLS0gQ3JlYXRlZCB3aXRoIElua3NjYXBlIChodHRwOi8vd3d3Lmlua3NjYXBlLm9yZy8pIC0tPgoKPHN2ZwogICB4bWxuczpkYz0iaHR0cDovL3B1cmwub3JnL2RjL2VsZW1lbnRzLzEuMS8iCiAgIHhtbG5zOmNjPSJodHRwOi8vY3JlYXRpdmVjb21tb25zLm9yZy9ucyMiCiAgIHhtbG5zOnJkZj0iaHR0cDovL3d3dy53My5vcmcvMTk5OS8wMi8yMi1yZGYtc3ludGF4LW5zIyIKICAgeG1sbnM6c3ZnPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICAgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIgogICB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIKICAgeG1sbnM6c29kaXBvZGk9Imh0dHA6Ly9zb2RpcG9kaS5zb3VyY2Vmb3JnZS5uZXQvRFREL3NvZGlwb2RpLTAuZHRkIgogICB4bWxuczppbmtzY2FwZT0iaHR0cDovL3d3dy5pbmtzY2FwZS5vcmcvbmFtZXNwYWNlcy9pbmtzY2FwZSIKICAgd2lkdGg9IjIwbW0iCiAgIGhlaWdodD0iMjBtbSIKICAgdmlld0JveD0iMCAwIDIwIDIwIgogICB2ZXJzaW9uPSIxLjEiCiAgIGlkPSJzdmc4IgogICBpbmtzY2FwZTp2ZXJzaW9uPSIwLjkyLjUgKDIwNjBlYzFmOWYsIDIwMjAtMDQtMDgpIgogICBzb2RpcG9kaTpkb2NuYW1lPSJjb25lLnN2ZyI+CiAgPGRlZnMKICAgICBpZD0iZGVmczIiPgogICAgPHJhZGlhbEdyYWRpZW50CiAgICAgICBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSIKICAgICAgIGdyYWRpZW50VHJhbnNmb3JtPSJtYXRyaXgoLTEuNjU5LDEuNzQxNSwtMS41MDcwNywtMS40MzU2Niw0MjUuNzUyLC05Mi44MikiCiAgICAgICByPSIzNC41NjgwMDEiCiAgICAgICBmeT0iNTQuODc5MDAyIgogICAgICAgZng9IjE0NS41NjU5OSIKICAgICAgIGN5PSI1NC44NzkwMDIiCiAgICAgICBjeD0iMTQ1LjU2NTk5IgogICAgICAgaWQ9ImUiCiAgICAgICB4bGluazpocmVmPSIjYSIgLz4KICAgIDxsaW5lYXJHcmFkaWVudAogICAgICAgaWQ9ImEiPgogICAgICA8c3RvcAogICAgICAgICBpZD0ic3RvcDExIgogICAgICAgICBzdG9wLWNvbG9yPSIjMDBjOGJkIgogICAgICAgICBvZmZzZXQ9IjAiIC8+CiAgICAgIDxzdG9wCiAgICAgICAgIGlkPSJzdG9wMTMiCiAgICAgICAgIHN0b3Atb3BhY2l0eT0iMCIKICAgICAgICAgb2Zmc2V0PSIxIiAvPgogICAgPC9saW5lYXJHcmFkaWVudD4KICAgIDxyYWRpYWxHcmFkaWVudAogICAgICAgeGxpbms6aHJlZj0iI2EiCiAgICAgICBpZD0iZiIKICAgICAgIGdyYWRpZW50VW5pdHM9InVzZXJTcGFjZU9uVXNlIgogICAgICAgZ3JhZGllbnRUcmFuc2Zvcm09Im1hdHJpeCgtMS45MjI5MSwtMS40NDQ4MiwxLjI1MDMxLC0xLjY2NDA2LDI5MC4yOSw1NTcuNzE3KSIKICAgICAgIGN4PSIxNDUuNTY1OTkiCiAgICAgICBjeT0iNTQuODc5MDAyIgogICAgICAgZng9IjE0NS41NjU5OSIKICAgICAgIGZ5PSI1NC44NzkwMDIiCiAgICAgICByPSIzNC41NjgwMDEiIC8+CiAgICA8bGluZWFyR3JhZGllbnQKICAgICAgIGdyYWRpZW50VHJhbnNmb3JtPSJtYXRyaXgoMC45MzU2MSwwLDAsMSw5LjE3MiwtMS4zMSkiCiAgICAgICBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSIKICAgICAgIHkyPSIxNjguMzc5IgogICAgICAgeDI9IjIxNi41MzQiCiAgICAgICB5MT0iMTY2Ljc0MSIKICAgICAgIHgxPSIxNjcuMDY5IgogICAgICAgaWQ9ImciCiAgICAgICB4bGluazpocmVmPSIjYyIgLz4KICAgIDxsaW5lYXJHcmFkaWVudAogICAgICAgaWQ9ImMiPgogICAgICA8c3RvcAogICAgICAgICBpZD0ic3RvcDIzIgogICAgICAgICBzdG9wLWNvbG9yPSIjZmZmZmJkIgogICAgICAgICBvZmZzZXQ9IjAiIC8+CiAgICAgIDxzdG9wCiAgICAgICAgIGlkPSJzdG9wMjUiCiAgICAgICAgIHN0b3Atb3BhY2l0eT0iMCIKICAgICAgICAgc3RvcC1jb2xvcj0iI2ZmZiIKICAgICAgICAgb2Zmc2V0PSIxIiAvPgogICAgPC9saW5lYXJHcmFkaWVudD4KICAgIDxsaW5lYXJHcmFkaWVudAogICAgICAgZ3JhZGllbnRUcmFuc2Zvcm09Im1hdHJpeCgxLDAsMCwxLjEzMjcsMS45MDIsLTI0LjYyOCkiCiAgICAgICBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSIKICAgICAgIHkyPSIxNjkuMTk4IgogICAgICAgeDI9IjE5OC4yODIiCiAgICAgICB5MT0iMTcxLjE0OSIKICAgICAgIHgxPSIyNDMuMjg0IgogICAgICAgaWQ9ImgiCiAgICAgICB4bGluazpocmVmPSIjZCIgLz4KICAgIDxsaW5lYXJHcmFkaWVudAogICAgICAgaWQ9ImQiPgogICAgICA8c3RvcAogICAgICAgICBpZD0ic3RvcDI5IgogICAgICAgICBzdG9wLWNvbG9yPSIjMDEyMjE3IgogICAgICAgICBvZmZzZXQ9IjAiIC8+CiAgICAgIDxzdG9wCiAgICAgICAgIGlkPSJzdG9wMzEiCiAgICAgICAgIHN0b3Atb3BhY2l0eT0iMCIKICAgICAgICAgb2Zmc2V0PSIxIiAvPgogICAgPC9saW5lYXJHcmFkaWVudD4KICAgIDxyYWRpYWxHcmFkaWVudAogICAgICAgZ3JhZGllbnRVbml0cz0idXNlclNwYWNlT25Vc2UiCiAgICAgICBncmFkaWVudFRyYW5zZm9ybT0ibWF0cml4KC0xLjAwOTgxOSwxLjA2MDAzNiwtMC45MTczNDA1NCwtMC44NzM4NzM5LDEzNy44OTE4OCwyNTMuMjgxMjIpIgogICAgICAgcj0iMzQuNTY4MDAxIgogICAgICAgZnk9IjU0Ljg3OTAwMiIKICAgICAgIGZ4PSIxNDUuNTY1OTkiCiAgICAgICBjeT0iNTQuODc5MDAyIgogICAgICAgY3g9IjE0NS41NjU5OSIKICAgICAgIGlkPSJlLTMiCiAgICAgICB4bGluazpocmVmPSIjYSIgLz4KICA8L2RlZnM+CiAgPHNvZGlwb2RpOm5hbWVkdmlldwogICAgIGlkPSJiYXNlIgogICAgIHBhZ2Vjb2xvcj0iI2ZmZmZmZiIKICAgICBib3JkZXJjb2xvcj0iIzY2NjY2NiIKICAgICBib3JkZXJvcGFjaXR5PSIxLjAiCiAgICAgaW5rc2NhcGU6cGFnZW9wYWNpdHk9IjAuMCIKICAgICBpbmtzY2FwZTpwYWdlc2hhZG93PSIyIgogICAgIGlua3NjYXBlOnpvb209IjIuNTYiCiAgICAgaW5rc2NhcGU6Y3g9IjI5NC41MzU4MyIKICAgICBpbmtzY2FwZTpjeT0iMTY2LjU2NzU5IgogICAgIGlua3NjYXBlOmRvY3VtZW50LXVuaXRzPSJtbSIKICAgICBpbmtzY2FwZTpjdXJyZW50LWxheWVyPSJnMTAzIgogICAgIHNob3dncmlkPSJmYWxzZSIKICAgICBpbmtzY2FwZTp3aW5kb3ctd2lkdGg9IjE5MjAiCiAgICAgaW5rc2NhcGU6d2luZG93LWhlaWdodD0iMTA0MyIKICAgICBpbmtzY2FwZTp3aW5kb3cteD0iMTkyMCIKICAgICBpbmtzY2FwZTp3aW5kb3cteT0iMCIKICAgICBpbmtzY2FwZTp3aW5kb3ctbWF4aW1pemVkPSIxIiAvPgogIDxtZXRhZGF0YQogICAgIGlkPSJtZXRhZGF0YTUiPgogICAgPHJkZjpSREY+CiAgICAgIDxjYzpXb3JrCiAgICAgICAgIHJkZjphYm91dD0iIj4KICAgICAgICA8ZGM6Zm9ybWF0PmltYWdlL3N2Zyt4bWw8L2RjOmZvcm1hdD4KICAgICAgICA8ZGM6dHlwZQogICAgICAgICAgIHJkZjpyZXNvdXJjZT0iaHR0cDovL3B1cmwub3JnL2RjL2RjbWl0eXBlL1N0aWxsSW1hZ2UiIC8+CiAgICAgICAgPGRjOnRpdGxlPjwvZGM6dGl0bGU+CiAgICAgIDwvY2M6V29yaz4KICAgIDwvcmRmOlJERj4KICA8L21ldGFkYXRhPgogIDxnCiAgICAgaW5rc2NhcGU6bGFiZWw9IkxpdmVsbG8gMSIKICAgICBpbmtzY2FwZTpncm91cG1vZGU9ImxheWVyIgogICAgIGlkPSJsYXllcjEiCiAgICAgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMCwtMjc3KSI+CiAgICA8ZwogICAgICAgaWQ9ImcxMDMiCiAgICAgICB0cmFuc2Zvcm09Im1hdHJpeCgwLjI2NDU4MzMzLDAsMCwwLjI2NDU4MzMzLDI4LjY2ODE3NSwxODkuNzQ3NzgpIj4KICAgICAgPHBhdGgKICAgICAgICAgaWQ9InBhdGgzOCIKICAgICAgICAgZD0ibSAtNTYuNDA1NTI4LDM1Mi4wMzEwNSBjIC0yLjA4MDUsMy44MTE2MiAtMC42NTEzLDguMTYwNzEgNC4wNjkxLDkuMjc2NDQgbCAtMTUuNzQxMywzMC4xNTQ1OSBjIC0yMS45MTE2OTcsLTEuOTg2MTYgLTMyLjQzNTk5MiwtMjMuMTA0NzIgLTIyLjY5NTA5NywtNDUuNzI4NTUgeiIKICAgICAgICAgaW5rc2NhcGU6Y29ubmVjdG9yLWN1cnZhdHVyZT0iMCIKICAgICAgICAgc3R5bGU9ImZpbGw6dXJsKCNlLTMpO3N0cm9rZTojMDAwMGY4O3N0cm9rZS13aWR0aDowIiAvPgogICAgPC9nPgogIDwvZz4KPC9zdmc+Cg==
```

**Review by offtherailz — CHANGES_REQUESTED** [review]
Works fine, only some suggestions

**Comment by tdipisa** [comment]
> Works fine, only some suggestions

also in FF?

File: web/client/translations/data.en-US.json (hunk 1 above)
Comment (MV88) [reply web/client/translations/data.en-US.json]: ok sure, thanks for being more specific

File: web/client/translations/data.it-IT.json [hunk 2]
```diff
@@ -91,7 +91,7 @@
             "groupProperties": "Proprietà del gruppo",
             "featureInfo": "Feature Info",
             "featureInfoFormatLbl": "Formato risposta interrogazioni su mappa",
-            "guideText": "Sono supportati Url o URI Base64. Inserisci $\\{properties.TUO_ATTRIBUTO\\} per ottenere il valore da un'attributo di una feature",
+            "guideText": "Nell'URL dell'immagine puoi utilizzare del testo come $\\{properties.YOUR_ATTRIBUTE\\}  per rendere parametrico l'URL, a seconda degli attributi della feature. Supporta sia URL che \"URI dati\" codificati in base64",
```
Comment (offtherailz) [inline web/client/translations/data.it-IT.json:94]:
```suggestion
            "guideText": "Nell'URL dell'immagine puoi utilizzare variabili come $\\{properties.YOUR_ATTRIBUTE\\}  per rendere parametrico l'URL a seconda degli attributi della feature. Supporta sia URL che \"Data URI\" codificati in base64",
```

File: web/client/translations/data.es-ES.json [hunk 3]
```diff
@@ -91,7 +91,7 @@
             "groupProperties": "Propiedades del grupo",
             "featureInfo": "Información de la capa",
             "featureInfoFormatLbl": "Identificar el formato de respuesta",
-            "guideText": "Admite URL o Base64URI. Utilice un marcador de posición como $\\{properties.YOUR_ATTRIBUTE\\} para obtener la URL de un atributo de característica",
+            "guideText": "En la URL de la imagen, puede utilizar marcadores de posición como $\\{properties.YOUR_ATTRIBUTE\\} para hacer que la URL sea paramétrica, según los atributos de la característica. Admite URL o \"URI de datos\" codificados en base64",
```
Comment (offtherailz) [inline web/client/translations/data.es-ES.json:94]:
```suggestion
            "guideText": "En la URL de la imagen, puede utilizar marcadores de posición como $\\{properties.YOUR_ATTRIBUTE\\} para hacer que la URL sea paramétrica, según los atributos de la característica. Admite URL o \"Data URI\\" codificados en base64",
```

File: web/client/translations/data.fr-FR.json [hunk 4]
```diff
@@ -91,7 +91,7 @@
             "groupProperties": "Propriétés du groupe",
             "featureInfo": "Informations attributaires de l'objet",
             "featureInfoFormatLbl": "Information, format de la réponse",
-            "guideText": "Prend en charge les URL ou Base64URI. Utilisez un espace réservé comme $\\{properties.YOUR_ATTRIBUTE\\} afin de récupérer l'URL d'un attribut de fonctionnalité",
+            "guideText": "Dans l'URL de l'image, vous pouvez utiliser des espaces réservés comme $\\{properties.YOUR_ATTRIBUTE\\} pour rendre l'URL paramétrique, en fonction des attributs de la fonctionnalité. Il prend en charge à la fois les URL ou les « URI de données» codés en base64.",
```
Comment (offtherailz) [inline web/client/translations/data.fr-FR.json:94]:
```suggestion
            "guideText": "Dans l'URL de l'image, vous pouvez utiliser des espaces réservés comme $\\{properties.YOUR_ATTRIBUTE\\} pour rendre l'URL paramétrique, en fonction des attributs de la fonctionnalité. Il prend en charge à la fois les URL ou les « Data URI » codés en base64.",
```

File: web/client/translations/data.de-DE.json [hunk 5]
```diff
@@ -91,7 +91,7 @@
             "groupProperties": "Gruppeneigenschaften",
             "featureInfo": "Feature Info",
             "featureInfoFormatLbl": "Antwortformat festlegen",
-            "guideText": "Unterstützt URLs oder Base64URI. Verwenden Sie einen Platzhalter wie $\\{properties.YOUR_ATTRIBUTE\\}, um die URL von einem Feature-Attribut abzurufen",
+            "guideText": "In der URL des Bildes können Sie Platzhalter wie $\\{properties.YOUR_ATTRIBUTE\\} verwenden, um die URL abhängig von den Feature-Attributen parametrisch zu machen. Es unterstützt sowohl URLs als auch „Daten-URIs“, die in Base64 codiert sind",
```
Comment (offtherailz) [inline web/client/translations/data.de-DE.json:94]:
```suggestion
            "guideText": "In der URL des Bildes können Sie Platzhalter wie $\\{properties.YOUR_ATTRIBUTE\\} verwenden, um die URL abhängig von den Feature-Attributen parametrisch zu machen. Es unterstützt sowohl URLs als auch „Data URI“, die in Base64 codiert sind",
```

**Review by offtherailz — APPROVED** [review]

**Comment by ElenaGallo** [comment]
Test passed, @MV88 please backport this to the stable branch. Thanks

**Pull request merged**